exporters/zipkin/build.gradle

```diff
@@ -19,11 +19,22 @@ dependencies {
     implementation libraries.zipkin_urlconnection
 
     testImplementation libraries.guava
-
+    testImplementation libraries.zipkin_junit
```

Comment:
nit: you can use comma separated here instead of having multiple `testImplementation`.